--- aqbanking-2.3.2/src/test/Makefile.am.orig	2006-04-08 10:03:17.000000000 +0200
+++ aqbanking-2.3.2/src/test/Makefile.am	2007-08-24 23:48:40.122284980 +0200
@@ -4,7 +4,7 @@ INCLUDES = -I$(top_builddir)/headers $(a
 noinst_PROGRAMS=abtest 
 
 abtest_SOURCES=abtest.c
-abtest_LDADD = -L$(top_builddir)/src/libs/aqbanking $(aqbanking_internal_libs) $(gwenhywfar_libs)
+abtest_LDADD = $(aqbanking_internal_libs) $(gwenhywfar_libs)
 
 #cpptest_SOURCES=cpptest.cpp
 #cpptest_LDADD = -L$(top_builddir)/src/libs/aqbanking $(aqbanking_internal_libs) $(top_builddir)/src/libs/aqbanking++/libaqbankingpp.la $(gwenhywfar_libs) -lstdc++
--- aqbanking-2.3.2/src/plugins/backends/aqyellownet/plugin/Makefile.am.orig	2007-01-12 22:30:06.000000000 +0100
+++ aqbanking-2.3.2/src/plugins/backends/aqyellownet/plugin/Makefile.am	2007-08-24 23:48:01.144063738 +0200
@@ -27,7 +27,7 @@ aqyellownet_la_SOURCES= \
  aqyellownet-plugin.c
 
 aqyellownet_la_LIBADD= \
-  -L$(srcdir) -laqyellownet \
+  $(builddir)/libaqyellownet.so \
   $(aqbanking_internal_libs) \
   $(gwenhywfar_libs)
 aqyellownet_la_LDFLAGS=-module -avoid-version
--- aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/logmanager/Makefile.am.orig	2006-04-08 10:04:14.000000000 +0200
+++ aqbanking-2.3.2/src/plugins/backends/aqhbci/ui/qt3/libs/logmanager/Makefile.am	2007-08-24 23:46:21.666394828 +0200
@@ -31,7 +31,7 @@ liblogmanager_la_SOURCES=\
   loganalyzer.cpp
 
 liblogmanager_la_LIBADD = \
-  -L$(top_builddir)/src/plugins/backends/aqhbci/plugin $(aqhbci_internal_libs)
+  $(aqhbci_internal_libs)
 
 BUILT_SOURCES = $(nodist_liblogmanager_la_SOURCES)
 CLEANFILES = $(nodist_liblogmanager_la_SOURCES) $(nodist_noinst_HEADERS)
--- aqbanking-2.3.2/src/plugins/bankinfo/generic/Makefile.am.orig	2006-09-21 19:52:09.000000000 +0200
+++ aqbanking-2.3.2/src/plugins/bankinfo/generic/Makefile.am	2007-08-24 23:46:00.221172735 +0200
@@ -17,7 +17,7 @@ noinst_HEADERS=\
 libbankinfo_la_SOURCES=\
  generic.c
 
-libbankinfo_la_LIBADD = -L$(top_builddir)/src/libs/aqbanking $(aqbanking_internal_libs) $(gwenhywfar_libs)
+libbankinfo_la_LIBADD = $(aqbanking_internal_libs) $(gwenhywfar_libs)
 libbankinfo_la_LDFLAGS = $(STRIPALL)
 
 at_files=at/blz.idx at/bic.idx at/namloc.idx at/banks.data
@@ -125,7 +125,7 @@ noinst_PROGRAMS=
 else
 noinst_PROGRAMS=mkdeinfo
 mkdeinfo_SOURCES=mkdeinfo.c
-mkdeinfo_LDADD = -L$(top_builddir)/src/libs/aqbanking $(aqbanking_internal_libs) $(gwenhywfar_libs)
+mkdeinfo_LDADD = $(aqbanking_internal_libs) $(gwenhywfar_libs)
 endif
 # IS_WINDOWS
 
