--- aqbanking-1.6.2/configure.ac.orig	2006-01-15 17:31:35.000000000 +0100
+++ aqbanking-1.6.2/configure.ac	2006-01-15 19:00:57.000000000 +0100
@@ -1343,15 +1343,14 @@
 
 case "$OS_TYPE" in
     posix)
-        libsuffix="`eval kde-config --libsuffix`"
-        kbanking_libs="-L\${libdir}${libsuffix} -lkbanking"
+        kbanking_libs="-L\${libdir} -lkbanking"
         kbanking_internal_libs="${abs_top_builddir}/src/frontends/kbanking/libkbanking.la"
         kbanking_includes="-I\${prefix}/include"
         kbanking_data="\${datadir}/kbanking"
     	;;
     windows)
         AC_DEFINE_UNQUOTED(BUILDING_KBANKING_DLL,1, [Define if DLL is built])
-        kbanking_libs="-L\${libdir}${libsuffix} -lkbanking32_${KBANKING_SO_EFFECTIVE}"
+        kbanking_libs="-L\${libdir} -lkbanking32_${KBANKING_SO_EFFECTIVE}"
         kbanking_internal_libs="-L${abs_top_builddir}/src/frontends/kbanking -lkbanking32_${KBANKING_SO_EFFECTIVE}"
         kbanking_includes="-I\${prefix}/include"
         kbanking_data="\${datadir}/kbanking"
